apps/server/src/modules/tool/service/school-external-tool.service.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@nestjs/common';
+import { SchoolExternalToolRepo } from '@shared/repo/schoolexternaltool/school-external-tool.repo';
+import { SchoolExternalToolDO } from '@shared/domain/domainobject/external-tool/school-external-tool.do';
+
+@Injectable()
+export class SchoolExternalToolService {
+	constructor(private readonly schoolExternalToolRepo: SchoolExternalToolRepo) {}
+
+	async findSchoolExternalTools(query: Partial<SchoolExternalToolDO>): Promise<SchoolExternalToolDO[]> {
+		const schoolExternalToolDOS: SchoolExternalToolDO[] = await this.schoolExternalToolRepo.find(query);
```

Comment:
Please pay attention to consistent naming (e.g. `schoolExternalToolDOs` instead of `schoolExternalToolDOS`).